src/toICN-after.js

```diff
@@ -36,10 +39,13 @@ if(detectedKey == ""){
   key = "";
   detectedKeyMinorSignature = "u";
 }
+else{
+  isKeyWritten = true;
+}
 
-let displayedKey = exports.DisplayedKey(detectedKey, detectedMinorSignature);
+let displayedKey = exports.getDisplayedKey(detectedKey, detectedKeyMinorSignature);
 // キーの手動設定
-var result = prompt("自動検出されたキー:" + displayedKey + "\n別のキーを指定したい場合は、下にキーを入力してください。(例:C)\nよくわからなければ、そのままOKを押してください。");
+var result = prompt("Key:" + displayedKey + isKeyWritten?"(Webサイトが指定したキー)":"(コード譜を元に自動判定されたキー)" +"\n別のキーを指定したい場合は、下にキーを入力してください。(例:C)\nよくわからなければ、そのままOKを押してください。");
```

Comment:
`(Webサイトが指定したキー)` とだけ出るようです。

```suggestion
var result = prompt("Key:" + displayedKey + (isKeyWritten?"(Webサイトが指定したキー)":"(コード譜を元に自動判定されたキー)") +"\n別のキーを指定したい場合は、下にキーを入力してください。(例:C)\nよくわからなければ、そのままOKを押してください。");
```